conf/httpd.conf.d/httpd.admin

```diff
@@ -100,8 +100,8 @@ if (defined($management_network->{'Tip'}) && $management_network->{'Tip'} ne '')
     $vhost = "0.0.0.0:".$PfConfig->{'ports'}{'admin'};
 }
 
-$User = "pf";
-$Group = "pf";
+$User = "pf_admin";
+$Group = "pf_admin";
```

Comment:
This will cause problem elsewhere.
We will not be able to modify the conf/*.conf files from the httpd.admin.
Since httpd.admin is not running as pf:pf anymore.

I think it is best to keep it a single user it is simpler.

However if you really want to go this route.
Do the following.
* Fix packaging there no where this pf_admin user is created
* Move to a separate sudoers file for each pf system user
* Make sure that conf/*.conf is read and writable from the httpd.admin
* Add the appropriate code to pfcmd fixpermissions
* Create the proper group permissions to segment who can do what